engine/src/main/resources/application.yml

```diff
@@ -1,3 +1,115 @@
+# suppress inspection "SpringBootApplicationProperties" for whole file
```

Comment:
What is the reason not to leverage Spring profiles as before? I assume we will do it differently

Additionally we got missed all commit history from the deleted files

Reply:
At this point, almost all the meaningful options have been removed from profiles in favor of being injected via docker environment variables.
So, these profiles are obsolete. A couple of timeouts specified for dev and integration profiles are not really important and can't justify keeping an extra source of configuration in already too complicated technology soup.

Fundamentally, spring profiles are not a good way to provide environment specific configuration, because it makes code dependent on the environments. Which is in general unsuitable for an open source, because code is public and environments are private.

Reply:
Yes, luckily 'application.yml' is not a brand-new file so you can still easily navigate to some of the commit history items through it. One day you show me how to get to commit history items through files which are no longer present, I am looking forward to it

Reply:
Be my guest: `git log -p -- engine/src/main/resources/`